packages/react/tasks/build.js

```diff
@@ -107,9 +114,10 @@ const babelConfig = {
     '@babel/plugin-transform-react-constant-elements',
   ],
   babelHelpers: 'bundled',
+  extensions: ['.ts', '.tsx', '.js', '.jsx'],
```

Comment:
Also not super familiar with this code, but just want to check if we need to consider any other extensions such as `.mjs`, `.cjs`, etc.

Reply:
@jdharvey-ibm I think it's not needed, since this task is specific to react package. I've copied the value provided in this line from the original https://github.com/carbon-design-system/carbon/blob/main/packages/react/tasks/build.js#L173 location. Also, related: https://github.com/babel/babel/issues/11577.